CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Add social button icon component `YoutubeButtonIcon` and add `YoutubeIcon` component.
```

Comment:
J'aurai simplement dit :
Add `YoutubeButtonIcon` and `YoutubeIcon`components.
Sinon, il faut reformuler comme suit :
Feature: Add social button icon `YoutubeButtonIcon` and add `YoutubeIcon` component.